src/SprykerEco/Zed/ComputopApi/Business/Mapper/ExpressCheckout/PayPalExpressCompleteMapper.php

```diff
@@ -53,19 +53,19 @@ protected function encryptRequestData(QuoteTransfer $quoteTransfer): array
     /**
      * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
      *
-     * @return array<string, string>
+     * @return array<string, int|string|null>
      */
     protected function getDataSubArray(QuoteTransfer $quoteTransfer): array
     {
-        $computopPayPalExpressPaymentTransfer = $quoteTransfer->getPayment()->getComputopPayPalExpress();
-
-        $dataSubArray[SharedComputopApiConfig::TRANS_ID] = $computopPayPalExpressPaymentTransfer->getTransId();
-        $dataSubArray[SharedComputopApiConfig::MERCHANT_ID] = $this->computopApiConfig->getMerchantId();
-        $dataSubArray[SharedComputopApiConfig::PAY_ID] = $computopPayPalExpressPaymentTransfer->getPayPalExpressPrepareResponse()->getPayID();
-        $dataSubArray[SharedComputopApiConfig::REF_NR] = $computopPayPalExpressPaymentTransfer->getRefNr();
-        $dataSubArray[SharedComputopApiConfig::AMOUNT] = $quoteTransfer->getTotals()->getGrandTotal();
-        $dataSubArray[SharedComputopApiConfig::CURRENCY] = $computopPayPalExpressPaymentTransfer->getCurrency();
+        $computopPayPalExpressPaymentTransfer = $quoteTransfer->getPaymentOrFail()->getComputopPayPalExpressOrFail();
```

Comment:
please make sure all transfers you use are defined at xml file
I'm not able to find this definition


`vendor/bin/spryker-dev-console dev:validate-module-transfers -m ModuleName` could be used

Reply:
Yes, it's because transfer.xml had filename computop.transfer.xml. Renamed it and fixed missing transfers & properties.